keyvi/tests/keyvi/dictionary/fsa/internal/memory_map_manager_test.cpp

```diff
@@ -33,23 +34,22 @@ namespace dictionary {
 namespace fsa {
 namespace internal {
 
-BOOST_AUTO_TEST_SUITE( MemoryMapManagerTests )
+BOOST_AUTO_TEST_SUITE(MemoryMapManagerTests)
 
-BOOST_AUTO_TEST_CASE( basic ) {
+BOOST_AUTO_TEST_CASE(basic) {
   size_t chunkSize = 1024 * 1024;
 
   boost::filesystem::path path = boost::filesystem::temp_directory_path();
-  path /= boost::filesystem::unique_path(
-      "dictionary-fsa-unittest-%%%%-%%%%-%%%%-%%%%");
+  path /= boost::filesystem::unique_path("dictionary-fsa-unittest-%%%%-%%%%-%%%%-%%%%");
   boost::filesystem::create_directory(path);
   MemoryMapManager m(chunkSize, path, "basic test");
 
   m.GetAddress(0);
-  char* testptr = (char*) m.GetAddress(2 * chunkSize);
+  char* testptr = reinterpret_cast<char*>(m.GetAddress(2 * chunkSize));
```

Comment:
Maybe I'm mistaken, but do we need `reinterpret_cast` ? I assume `static_cast` should be enough.

Reply:
Yes, static_cast should be enough here.